Sources/Common/Logger/TidalLoggable.swift

```diff
@@ -0,0 +1,8 @@
+import Logging
+
+public protocol TidalLoggable {
```

Comment:
Thank you for bringing up some valid points. Here are a some reasons why I believe maintaining an abstraction layer is still beneficial:
- While it's true that we're currently using the Apple library and there's no _immediate_ plan to change it, _future_ requirements or changes in the library's API could mean a switch. With an abstraction in place, we can minimize the impact of such changes on the rest of the codebase.
- By abstracting the logging functionality, we decouple the logic from the library's specific logging implementation. This promotes cleaner code and makes it easier to manage dependencies. If any part of the library changes, we only need to update the wrapper rather than every instance where logging is used.
- Having a unified logging interface ensures consistent usage across the project. It can enforce specific log formats, levels, and metadata standards, which might be harder to achieve if everyone uses the library directly.
- The wrapper can add additional functionality or utilities that the base library does not provide. For example, we could add custom log filtering, formatting, or integration with other monitoring tools within the wrapper.
- (minor IMO) An abstraction layer allows us to create mock implementations for testing purposes.
- By creating this abstraction in one Common module, which other modules already import, means that the modules don't need to import yet another dependency. This simplifies the dependency graph and makes the project structure cleaner and easier to manage.

While I understand the current dependency and setup, I believe these points highlight the long-term benefits of maintaining an abstraction. I'm open to discussing this further and finding the best approach.